src/java/org/apache/cassandra/locator/AbstractReplicationStrategy.java

```diff
@@ -299,6 +298,8 @@ public RangesAtEndpoint getPendingAddressRanges(TokenMetadata metadata, Collecti
 
     public abstract void validateOptions() throws ConfigurationException;
 
+    public abstract void maybeWarnOnOptions() throws ConfigurationException;
```

Comment:
Since the purpose of the method is just warn about stuff I guess it won't raise `ConfigurationException`